src/util/NumType.h

```diff
@@ -27,10 +27,12 @@
 #include "Eigen/Core"
 #include "sophus/sim3.hpp"
 #include "sophus/se3.hpp"
-
+#include <vector>
 
 namespace dso
 {
+template <class T, class Allocator =  Eigen::aligned_allocator<std::pair<const int,T>>   >
+using vectorA = std::vector< T, Allocator > ;
```

Comment:
Same comment as https://github.com/uzh-rpg/rpg_svo/pull/193#discussion_r108003101
Using
```
#include<Eigen/StdVector>
EIGEN_DEFINE_STL_VECTOR_SPECIALIZATION(VecX)
```
will be less intrusive.